Bitcode/Regression/sap_fct_splitting/mocks.c

```diff
@@ -0,0 +1,24 @@
+typedef unsigned long long uint64_t;
+typedef long long int64_t;
+typedef unsigned int uint32_t;
+typedef char* opaque_ptr;
+
+// Globals
+opaque_ptr _debugInfo = 0;
+opaque_ptr _sourceCodeObj = 0;
+
+// Helper functions
+void disrupt_optimizer() {}
```

Comment:
I might be missing something, but where is this function used?

Reply:
Yes, this is indeed not used. Removed.